docs/evm/evm-overview.md

```diff
@@ -0,0 +1,82 @@
+---
+title: EVM Overview
+sidebar_position: 1
+---
+
+# EVM Overview
+
+Initially developed in-house in NEAR, the Aurora EVM is the official EVM for the
+NEAR ecosystem. _It accomplishes a 1:1 experience with the Ethereum protocol
+including adopting ETH as the base currency_. Beyond the base scope, the EVM also allows
+for extra precompiles which allows for the EVM to interact with the NEAR blockchain.
+
+:::tip
+We implore you to interact through the EVM through MetaMask or Hardhat as we provide
+and identical experience through our RPC. However, you may also interact with it through NEAR
```

Comment:
```suggestion
and identical experiences through our RPC. However, you may also interact with it through NEAR
```

Reply:
This actually should be "provide an* identical experience".